pkg/endpoints/pipeline.go

```diff
@@ -292,20 +304,23 @@ func (r Resource) CreatePipelineRunImpl(pipelineRunData ManualPipelineRun, names
 		params = append(params, v1alpha1.Param{Name: "helm-secret", Value: pipelineRunData.HELMSECRET})
 	}
 	// PipelineRun yaml defines references to resources
-	newPipelineRunData := definePipelineRun(generatedPipelineRunName, namespace, serviceAccount, pipelineRunData.REPOURL, pipeline, v1alpha1.PipelineTriggerTypeManual, resources, params)
+	newPipelineRunData, err := definePipelineRun(generatedPipelineRunName, namespace, serviceAccount, pipelineRunData.REPOURL, pipeline, v1alpha1.PipelineTriggerTypeManual, resources, params)
+	if err != nil {
+		logging.Log.Errorf("there was a problem defining the pipeline run: %s", err)
```

Comment:
Should we fail/return an error here? If the PipelineRunData failed to be created, then there's no reason to proceed.